packages/venia-ui/lib/components/CartPage/PriceAdjustments/ShippingMethods/shippingRadios.css

```diff
@@ -0,0 +1,32 @@
+.root {
+    align-items: flex-start;
+    display: flex;
```

Comment:
Do we want to make the prices line up?

![Screen Shot 2020-02-05 at 1 40 36 PM](https://user-images.githubusercontent.com/13182778/73876733-1ca44300-481d-11ea-8db0-ba8e6f545d4b.png)

If so, we'll need to change some things around here, namely making the `root` container a `grid`.

Reply:
I think it pretty much looks like the mocks. This would be a UX call. @schensley !

Reply:
The latest mocks don't seem to have them aligned. I'll do a final review with @schensley and see if we want to make that change though.

Reply:
I think it's fine without aligning them. The font weight creates enough contrast.